lib/pdn/pdn.c

```diff
@@ -215,15 +221,47 @@ static void parse_cgev_apn_rate_ctrl(const char *notif)
 	}
 }
 
+#ifdef CONFIG_UNITY
+void on_cgev(const char *notif)
+#else
 static void on_cgev(const char *notif)
+#endif
 {
 	parse_cgev(notif);
 	parse_cgev_apn_rate_ctrl(notif);
 }
 
-NRF_MODEM_LIB_ON_INIT(modem_init_hook, on_modem_init, NULL);
+static void pdn_defaults_override(void)
+{
+	int err;
+
+	err = pdn_ctx_configure(0, CONFIG_PDN_DEFAULT_APN,
```

Comment:
Here we hardcode CID=0 for "initial PDP context" as in TS 27.007.
But there are networks do allow CID=1, for example, when estanblishing primary PDN.

Reply:
We can't do much about that, I suggest we take this discussion with the modem team.
From what I understand, the initial PDP context has `cid=0`, from spec.
The network has nothing to do with CID numbers, which are just a modem-side handle for a PDP context.

Reply:
Understood